package.json

```diff
@@ -7,14 +7,8 @@
     "@rails/actioncable": "^7.0.3",
     "@rails/activestorage": "^7.0.3",
     "@rails/ujs": "^7.0.3",
-    "@rails/webpacker": "5.4.2",
     "arctic_admin": "3.2.2",
```

Comment:
not sure but maytbe this package is no longer needed when using arctic_admin without webpacker, can you check? https://github.com/cprodhomme/arctic_admin

Reply:
same comment for other packages, I guess that without webpacker the `package.json` is no longer in use?

Reply:
I rather leave it there if we're going to use esbuild in the future

Reply:
hmmm ok, the thing is that I'm not sure esbuild needs them or if activeadmin+arcticadmin are going to use the traditional asset pipeline